src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java

```diff
@@ -89,7 +90,9 @@ public void refuteIfResultDoesNotMatch() throws IOException {
      */
     @Test
     public void refuteIfErrorDoesNotMatch() {
-        this.exception.expect(IllegalStateException.class);
+        this.exception.expectCause(
```

Comment:
@scristalli what's the reason for this change?

Reply:
@llorllale due to the wrapping in `IoChecked` done by `Unchecked`:
```
return (new IoChecked(this.origin)).value();
```

as opposed to the old `UncheckedScalar`,
```
return this.origin.value();
```

now the `IllegalStateException` exception raised in the test is wrapped inside a `RuntimeException`, because it is now thrown by `UncheckedText`:
```
throw new RuntimeException(error);
```